utils/email.js

```diff
@@ -1,58 +1,29 @@
 import dotenv from 'dotenv-safe';
 import nodemailer from 'nodemailer';
-import { getPledgeInviteById } from '../airtable/request';
+import Constants from '../Constants';
 
 dotenv.config(); // Set up environment variables
 
-const SENDER_NAME = ''; // specify the name of the person sending an email, for example: "Nick Wong ⚡️🔋"
-
 const transporter = nodemailer.createTransport({
   host: 'smtp.gmail.com',
-  port: 587,
-  secure: false, // true for 465, false for other ports
+  port: 465,
```

Comment:
what's the differencce between these 2 ports?

Reply:
one works through HTTPS vs HTTP

Reply:
i think LOL